db/migrate/20191023192329_create_versions.rb

```diff
@@ -0,0 +1,26 @@
+# This migration creates the `versions` table, the only schema PT requires.
+# All other migrations PT provides are optional.
+class CreateVersions < ActiveRecord::Migration[5.2]
```

Comment:
This migration should be in the module and not at the App level like we do with all of the other Projects and model should also be labeled as va_forms_versions or VAForms::Versions